docs/en/sql-reference/functions/array-functions.md

```diff
@@ -2118,6 +2118,113 @@ Result:
 └─────────────────────┘
 ```
 
+
+## randomSampleFromArray
+
+Returns a random sample from an input array. The number of elements in the sample is determined by the `sampleSize` argument. The function ensures that each element in the input array has an equal chance of being included in the sample.
```

Comment:
(I'd probably omit the last sentence, it is not wrong but it states the obvious.)